### ISSUE TITLE
Seq2seq with theano backend crashes on model.fit() with dropout

### ISSUE THREAD

**Issue by xkalash1** [farizrahman4u/seq2seq#161]
I went through #118 (which address mode.evaluate()) and related issues. However, I am still having issue with **model.fit()** using dropout. Here is the sample code @dajaj posted. It still gives me the MissingInputError. I am using theano 0.8.2 and the latest RecurrentShop and Seq2seq. Thanks a lot!

```
from seq2seq import Seq2Seq
import numpy as np

X = np.random.rand(10,20,128)
y = X

model = Seq2Seq(input_dim=128,output_dim=128,output_length=20,hidden_dim=128,dropout=0.1)
model.compile(loss='mse',optimizer='rmsprop',metrics=['accuracy'])
model.fit(X,y,nb_epoch=5)
```

**Comment by jfowkes** [comment]
I can confirm that I also get the MissingInputError with **model.fit()** using nonzero dropout with theano 0.8.2, the latest RecurrentShop and Seq2seq.

**Comment by fsonntag** [comment]
Same here with Theano 0.9

**Comment by oaomly** [comment]
Any updates on this?

**Comment by jjery2243542** [comment]
Have the same problem.

**Comment by tairoon1** [comment]
Is there any update yet?

**Comment by cheryl1215** [comment]
I just sucked it up and used raw tensorflow seq2seq (legacy). I believe tensorflow has just released a new seq2seq library.

**Comment by JiaqingFu** [comment]
I have the same problem,and  the problem has last for more than one year.
